content/bn/infrastructure-as-code.md

```diff
@@ -0,0 +1,17 @@
+---
+title: কোড হিসেবে পরিকাঠামো (Infrastructure as Code)
+status: Completed
+category: ধারণা
+---
+
+## এটা কি
+
+কোড হিসেবে পরিকাঠামো হল, অবকাঠামোর সংজ্ঞা এক বা একধিক ফাইল হিসেবে সংরক্ষণ করার অনুশীলন। এটি প্রথাগত মডেলকে প্রতিস্থাপন করে যেখানে পরিকাঠামোকে একটি পরিষেবা হিসেবে মানবিকভাবে প্রতিবিধান করা হয় সাধারণত সেল স্ক্রিপ্ট বা অন্যান্য কনফিগারেশন উপাদানের মাধ্যমে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+ক্লাউড নেটিভ পদ্ধতিতে অ্যাপ্লিকেশন তৈরি করার জন্য পরিকাঠামো নিষ্পত্তিযোগ্য এবং পুনরুৎপাদনযোগ্য হতে হবে। সাধারণত মানুষের হস্তক্ষেপ ছাড়াই এটি একটি স্বয়ংক্রিয় এবং পুনরাবৃত্তি যোগ্য উপায় [স্কেল](/scalability/) করতে পারে। মানবিক নজরদারি প্রতিক্রিয়াশীলতা এবং [ক্লাউড নেটিভ অ্যাপ্লিকেশনগুলির](/cloud-native-apps/) প্রয়োজনীয়তা পূরণ করতে পারে না। মানবিক কাঠামো পরিবর্তনগুলি উৎপাদনযোগ্য নয়, দ্রুত পরিমাপ সীমার মধ্যে চলে যায় এবং মাইক্রো কনফিগারেশন ভুলগুলি করে।
+
+## এটা কিভাবে সাহায্য করে
+
+সার্ভার , লোড ব্যালেন্স এবং সামনের গুলিকে কোড হিসেবে তথ্যকেন্দ্রে উপস্থাপনের মাধ্যমে এটি পরিকাঠামগত দলকে অনুমোদন করে সবার জন্য সত্যের একটি একক উৎস এবং একটি [CI](/continuous-integration/)/[CD](/continuous-delivery/) পাইপ লাইনের মধ্যে তথ্য পরিচালনা করতে। সঙ্গে ভার্সন নিয়ন্ত্রণ ও উন্নয়নের প্রকৌশল সংযুক্ত থাকে
```

Comment:
Kindly create a proper Sentence.

Reply:
@mitul3737 Thank You, sir. I have made some changes in this file and added the suggestion you gave me. Please check it whenever you are free. I'm extremely sorry for the mistakes. Thank you again.